README.md

```diff
@@ -154,8 +154,7 @@ class PoolAddress(DTO):
 from credmark.types import Address
 
 def run(self, input):
-   address = Address(wallet_adress)
-   address.checksum
+   address = Address(wallet_adress).checksum
```

Comment:
I wanted to show you can use the address (lowercase) and also get the checksum. Please undo this.

Reply:
Add the comments

```
from credmark.types import Address

def run(self, input):
    address = Address(wallet_adress) # address stores the lower case version
    address.checksum # .checksum property returns the checksum
```